ls1intum/Artemis
# [Enhancement] Hide the name of the student when downloading repos for Programming Assessment

### Checklist

### Motivation and Context


This PR enables the possibility to hide the students name when downloading the repo. For the manual assessment for programming exercises we then can assure that the assessor is not biased when downloading the repo.

### Description

- Added boolean check if student name needs to be hidden
- Added alternative folder name when student name needs to be hidden

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Assess a programming exercise
4. Click on `Download repo`and download the repo (test different options)
5. Downloaded repo should not contain information about the student

### Test Coverage

Nothing changed

### Screenshots



<img width="503" alt="image" src="https://user-images.githubusercontent.com/41108487/98521871-4d198180-2274-11eb-8c75-2e9eed9d299d.png">
